src/LogicApp/LogicApp/help/Get-AzIntegrationAccountAssembly.md

```diff
@@ -0,0 +1,159 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.LogicApp.dll-Help.xml
+Module Name: Az.LogicApp
+online version: https://docs.microsoft.com/en-us/powershell/module/az.logicapp/get-azintegrationaccountassembly
+schema: 2.0.0
+---
+
+# Get-AzIntegrationAccountAssembly
+
+## SYNOPSIS
+Gets an integration account assembly.
+
+## SYNTAX
+
+### ByIntegrationAccount (Default)
+```
+Get-AzIntegrationAccountAssembly -ResourceGroupName <String> -ParentName <String> [-Name <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByInputObject
+```
+Get-AzIntegrationAccountAssembly [-Name <String>] -ParentObject <IntegrationAccount>
```

Comment:
Nitpick - Right now the order of the parameters is not consistent - identity parameter should always be list from top level (resourcegroupname or parentobject/id) down to bottom level (name).  To fix this, please move the ParentObject and ParentResourceId parameter definitions in the cs file to be above the name parameter, rather than last in the file.  It's very annoying that this is how to fix the syntax, but right now that is the only way to do so.

Reply:
So I'll do this as an order

1. ResourceGroup
2. Parent
   1. ParentObject
   2. ParentId
   3. ParentName
3. ResourceName
4. Rest of Parameters

Reply:
Perfect!